WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -303,13 +303,17 @@ final class OrderListViewModel {
             })
 
             if !hasResults {
-                 print("0 transactions. Banner 1 shown")
-             } else if IPPresultsCount < Constants.numberOfTransactions {
-                 print("< 10 transactions within 30 days. Banner 2 shown")
-             } else if IPPresultsCount >= Constants.numberOfTransactions {
-                 print(">= 10 transactions within 30 days. Banner 3 shown")
-             }
+                print("0 transactions. Banner 1 shown")
+                return .IPP_COD
+            } else if IPPresultsCount < Constants.numberOfTransactions {
+                print("< 10 transactions within 30 days. Banner 2 shown")
+                return .IPP_firstTransaction
+            } else if IPPresultsCount >= Constants.numberOfTransactions {
+                print(">= 10 transactions within 30 days. Banner 3 shown")
+                return .IPP_powerUsers
+            }
```

Comment:
Are we ready to remove the debug print statements from here now?

Reply:
We are! Removed dacb798d49f10cc6ecc61651e876a19cc898cd88